lib/features/children/generosity_challenge/cubit/generosity_challenge_vpc_setup_custom.dart

```diff
@@ -0,0 +1,17 @@
+sealed class GenerosityChallengeVpcSetupCustom {
```

Comment:
So just for understanding/confirmation these are extra added states to the CommonCubit

Reply:
Yes, these are extra needed states indeed. If I were to compare it to say "ChildDetailsState":

ChildDetailsInitialState = BaseState.initial()
ChildDetailsFetchingState = BaseState.loading()
ChildDetailsFetchedState = BaseState.data()
ChildDetailsErrorState = BaseState.showSnackbar()

ChildTopUpSuccessState, ChildTopUpFundsErrorState, ChildEditGivingAllowanceSuccessState are what you would define in the custom class above.

Custom classes are not be re-used. They should be specific to a cubit/ screen. So it's more accurate to say "these are extra states added to the GenerosityChallengeVpcSetupCubit" rather than the "CommonCubit".

Lemme know if anything I explained here is still unclear!